### ISSUE TITLE
Long post titles may overlap with nav bar

### ISSUE THREAD

**Issue by ragesoss** [WikiEducationFoundation/WikiEduWebsite2015#15]
![screen shot 2015-07-01 at 5 18 55 pm](https://cloud.githubusercontent.com/assets/848483/8467654/fd1a7556-2015-11e5-9074-3a4f971d6514.png)

**Comment by abhay313** [comment]
Can I take this issue ?

**Comment by ragesoss** [comment]
Please choose only one issue at a time. Just leave a comment to let us know
which one, and let us know if you stop working on it later.

**Comment by abhay313** [comment]
Okay, I want to work on this issue.

**Comment by abhay313** [comment]
By following README file can I run it on my local machine?

**Comment by ragesoss** [comment]
Yes, you should be able to. This is a wordpress theme, so you'll need to install wordpress, then install and activate this theme, in order to replicate the issue. The readme should be correct in broad strokes, but may be out of date in terms of the JS build tools.
